### DISCUSSION TITLE
Some questions about the name of widget and other things

### DISCUSSION THREAD

**Discussion by heddxh** [tsujan/Kvantum#844] [General]
For some reasons, this vertical tab(list?) looks weird(no transparent and blur, focus tab no corner radius) using theme [Utterly Nord Light](https://www.pling.com/p/1917104/), so I want to change it myself. But I found I don't know the name of this widget. (The application in my screenshots is Gwenview)
![Screenshot_20221218_100433](https://user-images.githubusercontent.com/63106571/208276311-f5c5257a-a635-49f2-9332-859dc8c6218e.png)
![Screenshot_20221218_100355](https://user-images.githubusercontent.com/63106571/208276312-074c16bc-4026-4750-822c-31332b79fba6.png)

Secondly, there is a transparent part "behind" my blur part in all QT applications. 
![Screenshot_20221218_101026](https://user-images.githubusercontent.com/63106571/208278326-2184655f-0c20-4dec-92c1-da5155c3fb7e.png)

I guess it is window decoration's fault so I modify the `themerc`(for me it is [Utterly-roundrc](https://store.kde.org/p/1903823)), decrease the `PaddingBottom``PaddingTop``PaddingLeft``PaddingRight`. It does work----the transparent part is mostly covered by the blur part. But I want to know is how can I avoid the transparent part exactly? Or how can i fill the blur part wholy in the window? May the latter is more elegent and can do with Kvantum.

Thanks for your incredible work! I spent days reading the issues and posts here and your patience is memorable.
And sorry for my poor English(

**Comment by tsujan** [comment]
> But I found I don't know the name of this widget.

It depends on the app and its code. Most probably, it's controlled by `GenericFrame` and `ItemView` sections of the kvconfig file of the theme (and, of course, its corresponding SVG file).

> there is a transparent part "behind" my blur part in all QT applications.

It should be the window decoration you use. It can't be controlled by Kvantum. Sorry, I can't help you in that regard; you might want to ask its author.

I've made [BreezeEnhanced](https://github.com/tsujan/BreezeEnhanced), as a fork of KDE's Breeze decoration, especially for use with Kvantum.

**Reply by heddxh** [reply]
Wow your decoration is so great without and bug and blend with my theme, thank you!
And I should read the document more carefully to fix other bugs in the Nord theme I an using :)
Besides, I wish you Merry Christmas in advance!

**Reply by tsujan** [reply]
Happy to know you've found it useful.

Merry Christmas!
